src/Morphir/Elm/Frontend.elm

```diff
@@ -912,10 +912,11 @@ mapDeclarationsToValue sourceFile expose decls =
                                     |> Node.value
                                     |> Name.fromString
 
-                            valueDef : Result Errors (AccessControlled (Value.Definition SourceLocation SourceLocation))
+                            valueDef : Result Errors (AccessControlled (Documented (Value.Definition SourceLocation SourceLocation)))
                             valueDef =
                                 Node range function
                                     |> mapFunction sourceFile
+                                    |> Result.map (Documented "")
```

Comment:
I forgot to mention this but we should be extracting the Elm docs here. We already do this for types so you should be able to do mostly the same: https://github.com/finos/morphir-elm/blob/8ae877f1d937c1f6042f33970698312bd1d47c5f/src/Morphir/Elm/Frontend.elm#L782-L787